torch/quantization/__init__.py

```diff
@@ -12,6 +13,23 @@ def default_eval_fn(model, calib_data):
     for data in calib_data:
         model(data)
 
+default_loss_fn = torch.nn.MSELoss(reduction='sum')
+def default_train_fn(model, train_data, loss_fn=default_loss_fn, optimizer=None):
+    r"""
+    Default train function takes a torch.utils.data.Dataset and train the model
+    on the dataset
+    """
+    if not optimizer:
+        optimizer = torch.optim.Adam(model.parameters(), lr=0.001)
+    for i in range(10):
```

Comment:
this is absolutely random - why do we have it? and I don't think we should provide a training loop in this case at all

Reply:
I can move this to the test file

Reply:
Fully agree, we do not need this.